server/internal/app/i18n.go

```diff
@@ -0,0 +1,51 @@
+package app
+
+import (
+	"github.com/labstack/echo/v4"
+	i18nFS "github.com/reearth/reearth-cms/server/i18n"
+	"github.com/reearth/reearth-cms/server/internal/adapter"
+	"github.com/reearth/reearthx/i18n"
+	"github.com/reearth/reearthx/log"
+	"golang.org/x/text/language"
+)
+
+var (
```

Comment:
Translate https://github.com/reearth/reearthx/blob/main/rerror/error.go#L22 also:

```suggestion
var (
	// rerror
	_ = i18n.T("not found")
	_ = i18n.T("internal")
	_ = i18n.T("invalid params")
	_ = i18n.T("not implemented")

```